### PR TITLE
Added rnavar pipeline config for Munin

### PR DESCRIPTION
**Pull request by praveenraj2018**
Add a new munin config for RNAvar pipeline

### CODE REVIEW

File: conf/pipeline/rnavar/munin.config [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  // Specific nf-core/rnavar params
+ 
+  igenomes_ignore     = true
+  
+  // Genome references
+  fasta               = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa'
+  fasta_fai           = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.fai'
+  gtf                 = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf'
+  gene_bed		        = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.bed'
```
Comment (maxulysse) [inline conf/pipeline/rnavar/munin.config:17]:
```suggestion
  gene_bed		     = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.bed'
```

File: conf/pipeline/rnavar/munin.config [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  // Annotation settings
+  annotation_cache    = true
+  cadd_cache          = true
+  cadd_indels         = '/data1/cache/CADD/v1.4/InDels.tsv.gz'
+  cadd_indels_tbi     = '/data1/cache/CADD/v1.4/InDels.tsv.gz.tbi'
+  cadd_wg_snvs        = '/data1/cache/CADD/v1.4/whole_genome_SNVs.tsv.gz'
+  cadd_wg_snvs_tbi    = '/data1/cache/CADD/v1.4/whole_genome_SNVs.tsv.gz.tbi'
+  snpeff_cache        = '/data1/cache/snpEff/'
+  snpeff_db	          = 'GRCh38.99'
```
Comment (maxulysse) [inline conf/pipeline/rnavar/munin.config:36]:
```suggestion
  snpeff_db	         = 'GRCh38.99'
```

**Comment by maxulysse** [comment]
Check out https://github.com/nf-core/configs#adding-a-new-pipeline-specific-config
We need to add that to rnavar:
https://github.com/nf-core/configs#enabling-pipeline-specific-configs-within-a-pipeline

File: conf/pipeline/rnavar/munin.config (hunk 2 above)
Comment (praveenraj2018) [reply conf/pipeline/rnavar/munin.config]: Done!

File: conf/pipeline/rnavar/munin.config [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  cadd_indels         	= '/data1/cache/CADD/v1.4/InDels.tsv.gz'
+  cadd_indels_tbi    	= '/data1/cache/CADD/v1.4/InDels.tsv.gz.tbi'
+  cadd_wg_snvs        	= '/data1/cache/CADD/v1.4/whole_genome_SNVs.tsv.gz'
+  cadd_wg_snvs_tbi    	= '/data1/cache/CADD/v1.4/whole_genome_SNVs.tsv.gz.tbi'
+  snpeff_cache        	= '/data1/cache/snpEff/'
+  snpeff_db	        = 'GRCh38.99'
+  vep_cache           	= '/data1/cache/VEP/'
+  vep_genome          	= 'GRCh38'
+  vep_species         	= 'homo_sapiens'
+  vep_cache_version   	= '99'
```
Comment (maxulysse) [inline conf/pipeline/rnavar/munin.config:40]:
```suggestion
  // Genome references
  fasta                 = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa'
  fasta_fai             = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.fai'
  gtf                   = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf'
  gene_bed              = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.bed'

  // Known genome resources
  dbsnp                 = '/data1/references/annotations/GATK_bundle/dbsnp_146.hg38.vcf.gz'
  dbsnp_tbi             = '/data1/references/annotations/GATK_bundle/dbsnp_146.hg38.vcf.gz.tbi'
  known_indels          = '/data1/references/annotations/GATK_bundle/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz'
  known_indels_tbi      = '/data1/references/annotations/GATK_bundle/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi'

  // STAR index
  star_index            = '/data1/references/CTAT_GenomeLib_v37_Mar012021/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/star150bp.idx/'

  // Annotation settings
  annotation_cache      = true
  cadd_cache            = true
  cadd_indels           = '/data1/cache/CADD/v1.4/InDels.tsv.gz'
  cadd_indels_tbi       = '/data1/cache/CADD/v1.4/InDels.tsv.gz.tbi'
  cadd_wg_snvs          = '/data1/cache/CADD/v1.4/whole_genome_SNVs.tsv.gz'
  cadd_wg_snvs_tbi      = '/data1/cache/CADD/v1.4/whole_genome_SNVs.tsv.gz.tbi'
  snpeff_cache          = '/data1/cache/snpEff/'
  snpeff_db             = 'GRCh38.99'
  vep_cache             = '/data1/cache/VEP/'
  vep_genome            = 'GRCh38'
  vep_species           = 'homo_sapiens'
  vep_cache_version     = '99'
```

**Pull request merged**